packages/manager/apps/adp/README.md

```diff
@@ -0,0 +1,43 @@
+# manager-adp
```

Comment:
Shouldn't be `manager-analytics-data-platform`?

Using abbreviation isn't quit relevant. It's sounds more like an internal codename.

/cc @jleveugle @frenautvh @cbourgois @marie-j should we have to stick with abbreviation or using an explicit module name instead? Thanks!